source/includes/_errors.md.erb

```diff
@@ -8,7 +8,7 @@ Error Code | Meaning
 206        | Réponse incomplete - Un des fournisseurs de données a renvoyé une erreur, la réponse est incomplète (les valeurs concernées contiennent le message par défaut: `Donnée indisponible`)
 400        | Mauvaise requête -- Le format de la requête est incorrecte
 401        | Non autorisé -- Token invalide ou manquant
-403        | Interdit -- Votre jeton ne vous donne pas accès à cette ressource (cas particulier des entreprises [non diffusés à tous](#siren-et-siret-non-diffus-s-tous))
+403        | Interdit -- Votre jeton ne vous donne pas accès à cette ressource (cas particulier des entreprises [non diffusés](#entreprises-non-diffus-es))
```

Comment:
Je ne suis pas sûr de bien comprendre, on réalise un filtrage quelconque par jeton pour l'accès aux non diffusées ? Si oui le message est OK.

Moi j'avais compris qu'il y avait un seul endpoint pour les entreprises et les établissements et qu'on pouvait spécifier qu'on voulait les non diffusées avec un paramètre de requête, et qu'il n'y aurait pas de différences de droit d'accès entre "INSEE Entreprise" et "INSEE Entreprise non diffusées". 

Il y a donc bel et bien des droits d'accès différents ?

Reply:
Si tu demandes un non diffusables l'INSEE nous *renvoyait* un 403 avant. À l'époque on avait décidé de garder le 403 car il reflète le plus la réalité (plutôt qu'un 50X qui ne transmettrai pas l'idée que c'est un problème de droit)

Ici j'ai voulu garder le comportement, je trouve que le code 403 est le plus logique à renvoyer dans ce cas là.

Reply:
Je parle pas du code mais du message... Est-ce qu'on attribue des droits spécifiques aux jetons pour que les usagers puissent accéder aux non diffusées ? Si la réponse est non, le message d'erreur "Votre jeton ne vous donne pas accès à cette ressource" n'a aucun sens.

Reply:
Ça c'est le message standard qu'on renvoie quand c'est vrai.
C'est pour ça que j'ai mis entre parenthèse "cas particulier des entreprises non diffusées"

Reply:
Je trouve que le message n'a aucun sens... Les utilisateurs vont nous ouvrir des tickets pour nous demander de leurs octroyer les droits d'accès aux entreprises non diffusées, et on va leur dire "nan en fait mettez juste _with_non_diffusable=true_ en paramètre"...

Reply:
le message renvoyé par l'API c'est justement ça, mais le code c'est 403.
Dans la doc je ne me que le message d'erreur standard qu'on renvoie dans le cas général

Reply:
on change on renvoie "451 - Unavailable For Legal Reasons"